tick/dataset/serializer.py

```diff
@@ -0,0 +1,32 @@
+# License: BSD 3 clause
```

Comment:
This could be added as a utility to the dataset module

Reply:
Is it not?